<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>501</width>
     <height>381</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="startMenu">
    <property name="styleSheet">
     <string notr="true">z</string>
    </property>
    <widget class="QLabel" name="labelText">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>9</y>
       <width>276</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Geuss  Number Game!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptbStart">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>312</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>84</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_combox">
        <property name="text">
         <string>Set the  time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>20s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120s</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ptbExit">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>312</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStartGIF">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>411</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelText</zorder>
    <zorder>label_combox</zorder>
    <zorder>horizontalSpacer</zorder>
    <zorder>horizontalSpacer_2</zorder>
    <zorder>horizontalSpacer_3</zorder>
    <zorder>horizontalSpacer_4</zorder>
    <zorder>comboBox</zorder>
    <zorder>ptbStart</zorder>
    <zorder>horizontalSpacer_6</zorder>
    <zorder>horizontalSpacer_7</zorder>
    <zorder></zorder>
    <zorder>label_combox</zorder>
    <zorder>comboBox</zorder>
    <zorder>ptbExit</zorder>
    <zorder>labelStartGIF</zorder>
   </widget>
   <widget class="QWidget" name="GameMenu">
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt0">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>282</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt1">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>282</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt2">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>282</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt3">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>282</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt4">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>282</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>328</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt7">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>328</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt9">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>328</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt8">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>328</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt6">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>328</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtDel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>282</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtHint">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>328</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>hint</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="winOver">
    <widget class="QLabel" name="labelWinGIF">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>331</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>You are winning!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loseOver">
    <widget class="QLabel" name="labelLoseGIF">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Over Game!</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
